Library causes linking errors when used in another project

When the library is used in a project, the following linking error occurs:

```
Error: No implementations provided for the following modules:
         Unix referenced from .opam/4.06.1+multicore/lib/kcas/kcas.cmxa(Kcas_backoff)
make: *** [bench] Error 1
```

This is due to Unix not being listed as a dependency in the kcas package metadata, so `unix.cmxa` is not linked before `kcas.cmxa`, despite it being required.

--- a/pkg/META
+++ b/pkg/META
@@ -1,6 +1,6 @@
 description = "Multi-word compare-and-swap library"
 version = "%%VERSION_NUM%%"
-requires = ""
+requires = "unix"
 archive(byte) = "kcas.cma"
 archive(native) = "kcas.cmxa"
 plugin(byte) = "kcas.cma"

